lib/wallet/src/Cardano/Api/Gen.hs

```diff
@@ -1399,9 +1399,9 @@ genUpdateProposal era =
         Nothing ->
             pure TxUpdateProposalNone
         Just supported ->
-            oneof
-                [ pure TxUpdateProposalNone
-                , TxUpdateProposal supported
+            frequency
+                [ (95, pure TxUpdateProposalNone)
+                , (5, TxUpdateProposal supported
```

Comment:
These are expensive to generate and really uninteresting to us.

They do affect the size. But `TxMetadata` is a cheaper thing which is can be big. Besides, testing balanceTx with arbitrary `maxTxSize` would be more effective, and we should also test `selectAssets` for arbitrary `requestedBalance` separately.